### ISSUE TITLE
Allow purchase_link button colors to override all other button colors

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/vendd#202]
I think it's a silly enhancement, but it should probably work.

**Comment by SDavisMedia** [comment]
Quick background of this issue is that a user wants to use Vendd's EDD button color picker setting but also override that color using `[purchase_link]` color attribute. So the proposed hierarchy would be EDD style settings for buttons are global, Vendd can override that (it falls back to EDD styles if not overwritten), and then have `[purchase_link]` override everything.

That said, I don't think this is going to work. No matter what, output of `[purchase_link]` is going to add color classes to the purchase button. Doesn't matter if you use the color attribute or not... at a minimum, `blue` will be there. And that will simply change if you use the attribute to display a different color.

The problem is, any CSS I write that allows the `[purchase_link]` color attribute to override Vendd's EDD button color setting and EDD's style settings is going to end up overriding all instances of the purchase buttons because the markup is identical whether it's `[purchase_link]` output, download details widget, filtered download content output, or anything else.

Bottom line, this might just have to be a `wontfix` unless there changes in EDD itself which would be adding a unique class to the `[purchase_link]` buttons just to indicate that the button is being output by the shortcode.

Still thinking lol.

**Comment by SDavisMedia** [comment]
Never mind. Got it working. :) It requires that EDD's button style setting is set to `inherit`. If set to anything else, the priority flow goes back to what it was. I can dig it!
